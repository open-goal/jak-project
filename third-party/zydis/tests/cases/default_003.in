-64 b7a7