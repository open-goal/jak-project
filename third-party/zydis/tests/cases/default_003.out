== [    BASIC ] ============================================================================================
   MNEMONIC: mov [ENC: DEFAULT, MAP: DEFAULT, OPC: 0xB7]
     LENGTH:  2
        SSZ: 64
       EOSZ:  8
       EASZ: 64
   CATEGORY: DATAXFER
    ISA-SET: I86
    ISA-EXT: BASE
 EXCEPTIONS: NONE
  OPTIMIZED: B7 A7 

== [ OPERANDS ] ============================================================================================
##       TYPE  VISIBILITY  ACTION      ENCODING   SIZE  NELEM  ELEMSZ  ELEMTYPE                        VALUE
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------
 0   REGISTER    EXPLICIT       W        OPCODE      8      1       8       INT                           bh
 1  IMMEDIATE    EXPLICIT       R         UIMM8      8      1       8      UINT  [U A  8] 0x00000000000000A7
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------

== [      ATT ] ============================================================================================
   ABSOLUTE: mov $0xA7, %bh
   RELATIVE: mov $0xA7, %bh

== [    INTEL ] ============================================================================================
   ABSOLUTE: mov bh, 0xA7
   RELATIVE: mov bh, 0xA7

== [ SEGMENTS ] ============================================================================================
B7 A7 
:  :..IMM
:..OPCODE
